index.html

```diff
@@ -257,7 +272,7 @@ <h3 class="skills_dropdown_title">
                     </div>
                     <ul class="skills_dropdown_desc">
                       <li class="skills_desc_li">
-                        <img src="images/Javascript.png" alt="javaScript"/>
+                        <img src="images/javascript.png" alt="javaScript"/>
```

Comment:
- [ ] Kindly please align these language items to the left, like the rest of the surrounding elements:

![image](https://user-images.githubusercontent.com/57421823/149132005-0b595031-14fb-4f35-95e0-2d4d28de36d3.png)

The figma:

![image](https://user-images.githubusercontent.com/57421823/149132063-35f597ed-c305-4ca6-b015-031fa2fe7cf0.png)

That way, the alignment will be a lot more consistent 💪

Reply:
Thanks for thoroughly reviewing this project. I fully understand every change that needs to be made. Thanks a lot 😉